product_seller/models/stock_quant.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Quartile
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class StockQuant(models.Model):
+    _inherit = "stock.quant"
+
+    seller_id = fields.Many2one(related="product_id.seller_id")
```

Comment:
No `store=True`?

Reply:
There is no reason to set store=false, so I added store=true.